tests/dummy/app/mirage/config.js

```diff
@@ -1,3 +1,89 @@
 export default function () {
+  this.get('/nodes', function () {
+    return {
```

Comment:
Just an FYI you can shove the data into `tests/dummy/app/mirage/fixtures/nodes.js` and change this to be:
```js
this.get('/nodes', function (db) {
  return {
    data: db.nodes
  }
}
```